pkg/issuer/acme/setup.go

```diff
@@ -105,14 +155,31 @@ func (a *Acme) Setup(ctx context.Context) (issuer.SetupResponse, error) {
 		glog.V(4).Infof("%s: %s", a.issuer.GetObjectMeta().Name, s)
 		a.Recorder.Event(a.issuer, v1.EventTypeWarning, errorAccountVerificationFailed, s)
 		a.issuer.UpdateStatusCondition(v1alpha1.IssuerConditionReady, v1alpha1.ConditionFalse, errorAccountRegistrationFailed, s)
+
+		acmeErr, ok := err.(*acmeapi.Error)
+		// If this is not an ACME error, we will simply return it and retry later
+		if !ok {
+			return issuer.SetupResponse{Requeue: true}, err
+		}
+
+		// If the status code is 400 (BadRequest), we will *not* retry this registration
+		// as it implies that something about the request (i.e. email address or private key)
+		// is invalid.
+		// TODO: What about other types of 4xx error?
+		if acmeErr.StatusCode == 400 {
+			glog.V(4).Infof("Skipping retrying account registration as a BadRequest response was returned from the ACME server: %v", acmeErr)
+			return issuer.SetupResponse{}, nil
+		}
+
+		// Otherwise if we receive anything other than a 400, we will retry.
 		return issuer.SetupResponse{Requeue: true}, err
 	}
 
 	glog.Infof("%s: verified existing registration with ACME server", a.issuer.GetObjectMeta().Name)
 	a.issuer.UpdateStatusCondition(v1alpha1.IssuerConditionReady, v1alpha1.ConditionTrue, successAccountRegistered, messageAccountRegistered)
 	a.issuer.GetStatus().ACMEStatus().URI = account.URL
 
-	return issuer.SetupResponse{}, err
+	return issuer.SetupResponse{}, nil
```

Comment:
This change is not really related to this PR, and is a no-op as we already handle the case where `err != nil` above 😄